creator-node/src/snapbackSM/snapbackSM.js

```diff
@@ -846,31 +794,6 @@ class SnapbackSM {
         time: Date.now()
       })
 
-      // Setup the mapping of Content Node endpoint to service provider id. Used in reconfig
```

Comment:
moved higher up in flow as this.peerSetManager.endpointToSpIdMap` is used in `aggregateReconfigAndPotentialSyncOps`